solutions/python-igraph/visualize_cliques/visualize_cliques_with_edges.py

```diff
@@ -0,0 +1,27 @@
+import igraph as ig
+import matplotlib.pyplot as plt
+
+g = ig.Graph.Famous('Zachary')
+g.vs['label'] = [i + 1 for i in range(g.vcount())]
+g.vs['size'] = 0
+g.es['width'] = 0.3
+g.vs['color'] = 'yellow'
+
+fig, ax = plt.subplots(3, 4)
+cliques = g.cliques(4, 4)
+for i in range(len(cliques)):
+    g.vs[cliques[i]]['size'] = 0.5
+    for j in range(len(cliques[0]) - 1):
+        for k in range(j + 1, len(cliques[0])):
```

Comment:
Instead of looping through all vertices and finding all the edge ids you can immediately select all relevant edges using
[`g.es.select`](https://igraph.org/python/api/latest/igraph.EdgeSeq.html#select), using the `_within` argument. That way, you can simply do

```
clique_edges = g.es.select(_within=clique)
clique_edges['color'] = 'red'
clique_edges['width'] = 1
```